fe1-web/src/features/wallet/screens/WalletSetSeed.tsx

```diff
@@ -28,7 +28,10 @@ const WalletSetSeed = ({ navigation }: IPropTypes) => {
   const initWallet = async () => {
     try {
       await Wallet.importMnemonic(seed);
-      navigation.navigate(STRINGS.navigation_synced_wallet);
+      navigation.reset({
```

Comment:
I would recommend you rely on `useNavigation()` instead of passing it through prop-types, it's just so much simpler 😉

Reply:
Ook !